Vagrantfile

```diff
@@ -8,6 +8,7 @@ Vagrant.configure('2') do |config|
   config.ssh.private_key_path = '~/.vagrant.d/insecure_private_key'
 
   config.vm.define 'anxs' do |machine|
+    #machine.vm.box = "generic/ubuntu1804"
```

Comment:
The addition of Ubuntu 18 is welcome, but it's probably best to do that as a separate PR rather than mixing it with the v11 update.

Reply:
Could you also make sure you update the Vagrant files, so that the new version gets tested.  This PR was only run against:

 ```
846.1
 Python: 2.7
 IMAGE_NAME="ubuntu-upstart:14.04"
 7 min 10 sec

 846.2
 Python: 2.7
 IMAGE_NAME="ubuntu:16.04-builded"
 6 min 21 sec

 846.3
 Python: 2.7
 IMAGE_NAME="debian:8-builded"
 7 min 30 sec

 846.4
 Python: 2.7
 IMAGE_NAME="debian:9-builded"
 6 min 6 sec

 846.5
 Python: 2.7
 IMAGE_NAME="centos:7-builded"
 5 min 10 sec

 846.6
 Python: 2.7
 IMAGE_NAME="centos:6-builded"
 21 min 36 sec
```